manifests/init.pp

```diff
@@ -315,6 +315,7 @@
     'x86_64', 'amd64': { $real_arch = 'amd64' }
     'i386':            { $real_arch = '386' }
     'aarch64':         { $real_arch = 'arm64' }
+    'arm64':           { $real_arch = 'amd64' }
```

Comment:
this looks wrong. If facter reports arm64 you want to have amd64? Isn't arm64 the equivalent to aarch64?

Reply:
thanks for the input! Initially I tried `'arm64':           { $real_arch = 'arm64' }` which didn't work back then. But it works now.

Reply:
I've just checked it again, but the `node_exporter` compiled for arm is still failing to start:
```
ci-macos-build-10:bin developer$ file node_exporter
node_exporter: Mach-O 64-bit executable arm64
ci-macos-build-10:bin developer$ ./node_exporter -h
Killed: 9
```

(I also tried to run the [node_exporter-1.6.1.darwin-arm64.tar.gz]([node_exporter-1.6.1.darwin-arm64.tar.gz](https://github.com/prometheus/node_exporter/releases/download/v1.6.1/node_exporter-1.6.1.darwin-arm64.tar.gz) directly with the same result).

So the only way for us to run it is to install/start it in Rosetta mode, therefor we need to stay with `amd64`